frontend/src/pages/CourseDetails/CourseDetails.module.css

```diff
@@ -0,0 +1,167 @@
+.courseDetailsContainer {
+  background-color: #fff;
+  padding: 20px;
+  margin: 20px auto;
```

Comment:
nit: I generally recommend using `em` or `rem` for padding and margin, see https://www.youtube.com/watch?v=N5wpD9Ov_To